phabricator.py

```diff
@@ -37,8 +39,25 @@ def run(self):
         # Find the preselected branch
         git_branch = settings.get('branch')
 
-        # If no preselected branch is provided
-        if git_branch == None:
+        if git_branch is None and settings.get('use_arc_land_onto_default', False):
```

Comment:
Nah, we can leave the old syntax as is. But for one-off options, it would be nice to encapsulate them. We can use something like:

```python
if isinstance(branch, basestr):
  pass
if isinstance(branch, dict):
  pass
else:
  pass
```